### PR TITLE
HTML layout for prototype

### PR DESCRIPTION
**Pull request by CarolineHana**
image
Updated image of how webpage looks

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  </head>
+  <body >
+      
+      <!-- content that resizes when side bar opens -->
+      <div id = "main">
+           <div class= "header" >
+                <img
+                 src="https://firebasestorage.googleapis.com/v0/b/codesign-prod.google.com.a.appspot.com/o/user_assets%2FlyxSOzzXdt1gu3M1bgLd%2FfGwBG6ucZZcvCT6IQ54Y%2Foriginal.svg?alt=media&token=2e67a119-d7c8-4cbb-91de-04f256f74609"
+                class = "YTimage" />
+                <span class = "title">Lounge</span>
```
Comment (krowngold) [inline src/main/webapp/index.html:20]: Can we change the title to "YouTube Lounge" instead?

Comment (CarolineHana) [reply src/main/webapp/index.html:20]: I used the youtube logo, its the image right above it

Comment (CarolineHana) [reply src/main/webapp/index.html:20]: or do you mean change the class name?

Comment (krowngold) [reply src/main/webapp/index.html:20]: My apologies, I misread the file and assumed you were creating a title element which in that case I wanted it to read as "YouTube Lounge" instead, but you already have that. Sorry!

**Comment by CarolineHana** [comment]
@googlebot I signed it!

**Comment by CarolineHana** [comment]
@googlebot I signed it!

File: src/main/webapp/index_script.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+//opens chat and queue on the side and adjusts content size on screen
```
Comment (ciddu) [inline src/main/webapp/index_script.js:1]: Can you fix the lint errors in this file.

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+           <div class= "header" >
+                <img
+                 src="https://firebasestorage.googleapis.com/v0/b/codesign-prod.google.com.a.appspot.com/o/user_assets%2FlyxSOzzXdt1gu3M1bgLd%2FfGwBG6ucZZcvCT6IQ54Y%2Foriginal.svg?alt=media&token=2e67a119-d7c8-4cbb-91de-04f256f74609"
+                class = "YTlogo" />
+                <span class = "title">Lounge</span>
+            </div> 
+        <button id="sidebarBtn" onclick="openSidebar()"> <span class="glyphicon glyphicon-menu-left"></span></button> 
+          <!-- video placeholder --> 
+          <div id = "videoplayer">
+            <iframe src="https://www.youtube.com/embed/y0U4sD3_lX4"></iframe>
```
Comment (clayferg) [inline src/main/webapp/index.html:25]: Could you add id="ytplayer" to the iframe element. It'll help me do the pause/play commands.

**Comment by CarolineHana** [comment]
@googlebot I signed it!

**Pull request merged**